widgets/bigquery-cdcTarget.json

```diff
@@ -25,7 +20,17 @@
       ]
     },
     {
-      "label" : "CDC Configuration",
+      "label": "Credentials",
+      "properties": [
+        {
+          "name": "serviceAccountKey",
+          "label": "Service Account Key",
+          "widget-type": "password"
```

Comment:
this is supposed to be a secret though, since it is the account key. In the cloud, we expect this to be left blank. In on-prem, we would normally expect this to be a secure macro.